libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -73,6 +78,17 @@ class NavigationRoute internal constructor(
             )
         }
 
+        internal suspend fun createEx(
```

Comment:
Maybe? 
```suggestion
        internal suspend fun createWithParallelParsing(
```

Reply:
Also let's make this function public!

Reply:
I thought we weren't exposing public suspend functions. Has that changed?

Reply:
I don't like this name. The method name shouldn't expose an internal implementation detail IMO.  The name I used isn't great either. I"m open to suggestions.

Reply:
I see. If we were able to change API, I would suggest names `create` for suspend function and `createBlocking` for blocking function. I guess it's a regular name conventions for kotlin coroutines code. But we can't change API. The blocking create should stay `create`. What if we add `Async` postfix to suspend function: `createAsync`. I haven't seen `Async` postfix in kotlin code yet but I can't imagine a name which makes more sense 🙂

Reply:
Opted to keep it internal for now, let's consider exposing separately.